### DISCUSSION TITLE
Is there a way to ignore modifier keys when configuring button overrides?

### DISCUSSION THREAD

**Discussion by Ardhimas** [linearmouse/linearmouse#583] [Q&A]
Simply put, I have a mouse where I swap the left click and the right click. However, if I need to hit `Cmd + Left Click`, the `Cmd` modifier will cause the override for `Left Click` to be ignored, thus reverting it back to `Cmd + Right Click`. Is there a way to set the behavior so that `Left Click` and `Right Click` remain swapped even with modifiers?

I tried to simply record all of the possible permutations, and it looks like you are able to do so for keyboard shortcuts, but not for mouse + keyboard shortcuts, as shown in the type error screenshot below.
![image](https://github.com/linearmouse/linearmouse/assets/5238654/54f36c63-2a10-4abe-83b3-a8dbe97bdf38)
![image](https://github.com/linearmouse/linearmouse/assets/5238654/82b845e2-7fdd-4615-b88d-799826ba42dc)

Thanks in advance.

**Comment by lujjjh** [comment]
Are you looking for a way to swap the functions of the left click and the right click for a particular device?

The issue you mentioned does exist. Currently, LinearMouse does not support ignoring modifier keys when matching button mappings. Also, LinearMouse does not support setting modifier keys for mouse button click actions. Both features need to be added.

To begin, I will add an option to switch the left click and right click, as it appears to be a commonly used option. Will this fulfill your current requirements?
